liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/CreateServerMojo.java

```diff
@@ -186,10 +186,19 @@ private String getApplicationFilename() {
         case "eba":
         case "esa":
             name += "." + project.getPackaging();
+            if (looseConfig) {
+                name += ".xml";
+            }
             break;
         case "liberty-assembly":
             // assuming liberty-assembly project will also have a war file output.
-            name += ".war";
+            File dir = new File(project.getBasedir() + "/src/main/webapp");
+            if (dir.exists()) {
+                name += ".war";
+                if (looseConfig) {
+                    name += ".xml";
+                }
+            }
```

Comment:
I know it didn't change, but can you fix up the debug message below:
 log.debug("The project artifact cannot be installed to a Liberty server because " + 
                     project.getPackaging() + " is not a supported packaging type.");